.vscode/settings.json

```diff
@@ -10,5 +10,6 @@
 	//Use consistent indentation
 	"editor.detectIndentation": false,
 	"editor.tabSize": 4,
-	"editor.insertSpaces": false
+	"editor.insertSpaces": false,
+	"java.compile.nullAnalysis.mode": "automatic",
```

Comment:
Did you mean to update this?

Reply:
No that sounds good! Just wanted to make sure that it was intentional. Other than that this PR looks good to me :+1: